src/test/java/Entities/PropertyTest.java

```diff
@@ -77,71 +77,71 @@ public void testSetOwner() {
     public void testGetName() {
         Player player1 = new Player("Player1");
         Property property1 = new Property("property1", "Red",
-                5,8, new int[]{1, 2, 3, 4, 5}, player1,200,2,true);
+                5,8, new int[]{1, 2, 3, 4, 5, 6}, player1,200,2,true);
         assertEquals("property1", property1.getName());
     }
 
     @Test
     public void testGetColour() {
         Player player1 = new Player("Player1");
         Property property1 = new Property("property1", "Red",
-                5,8, new int[]{1, 2, 3, 4, 5}, player1,200,2,true);
+                5,8, new int[]{1, 2, 3, 4, 5, 6}, player1,200,2,true);
         assertEquals("Red", property1.getColour());
     }
 
     @Test
     public void testGetPrice() {
         Player player1 = new Player("Player1");
         Property property1 = new Property("property1", "Red",
-                5,8, new int[]{1, 2, 3, 4, 5}, player1,200,2,true);
+                5,8, new int[]{1, 2, 3, 4, 5, 6}, player1,200,2,true);
         assertEquals(5, property1.getPrice());
     }
 
     @Test
     public void testGetHousesRent() {
         Player player1 = new Player("Player1");
         Property property1 = new Property("property1", "Red",
-                5,8, new int[]{1, 2, 3, 4, 5}, player1,200,2,true);
+                5,8, new int[]{1, 2, 3, 4, 5, 6}, player1,200,2,true);
         assertEquals(3, property1.getRent());
     }
 
     @Test
     public void testGetRentRailroad() {
         Player player1 = new Player("Player1");
         Property property1 = new Property("property1", "Railroad",
-                5,8, new int[]{1, 2, 3, 4, 5}, player1,200,2,true);
+                5,8, new int[]{1, 2, 3, 4, 5, 6}, player1,200,2,true);
         assertEquals(100, property1.getRent());
     }
 
     @Test
     public void testGetRentUtility() {
         Player player1 = new Player("Player1");
         Property property1 = new Property("property1", "Utility",
-                5,8, new int[]{1, 2, 3, 4, 5}, player1,200,2,true);
+                5,8, new int[]{1, 2, 3, 4, 5, 6}, player1,200,2,true);
```

Comment:
good catch here since in the property constructor there is a for loop from index 0 - 5, so we need 6 values here.